server/index.js

```diff
@@ -251,10 +257,26 @@ async function transformer(html, req) {
         '\\u003c'
       )};
       </script>`
+    )
+    .replace(
+      '<!--scripts-->',
+      `
+        <script src="https://www.unpkg.com/systemjs@6.1.7/dist/system.js"></script>
+        <script src="https://www.unpkg.com/systemjs@6.1.7/dist/extras/named-exports.js"></script>
+        <script type="systemjs-importmap">
+        {
+          "imports": {
+            "react": "https://unpkg.com/react@16.12.0/umd/react.production.min.js",
+            "react-dom": "https://unpkg.com/react-dom@16.12.0/umd/react-dom.production.min.js"
+          }
+        }
+      </script>
+    `
     );
 
   return dom;
 }
+{/* <script src="https://unpkg.com/browse/antd@4.9.1/dist/antd.min.js"></script> */}
```

Comment:
Why are we commenting this out? Looks important to me that we pass antd to plugins

Reply:
plugins are self contained, the antd lib is bundled within the plugins

Reply:
Okay, sounds good to me.